tests/controllers/qos_controller_test.cpp

```diff
@@ -37,8 +41,13 @@ TEST(SerenityControllerTest, NoQoSCorrections) {
   Try<Nothing> initialize = controller->initialize(
       lambda::bind(&MockSlaveUsage::usage, &usage));
 
+  process::Clock::pause();
```

Comment:
Can you elaborate on what you are trying to do here with the pause() and settle()?